src/pytest_qgis/qgis_bot.py

```diff
@@ -81,9 +86,34 @@ def create_feature_with_attribute_dialog(
         dialog = QgsAttributeDialog(
             layer, new_feature, False, self._iface.mainWindow(), True, context
         )
+        dialog.show()
         dialog.setMode(QgsAttributeEditorContext.AddFeatureMode)
 
-        # Two accepts to bypass some warnings
+        utils.wait(show_dialog_timeout_milliseconds)
+
+        warning_labels = [
+            label for label in dialog.findChildren(QLabel) if "#FFC107" in label.text()
```

Comment:
Should color codes be configurable somehow if changed in future QGIS version? So it's easier to maintain

Reply:
Good idea, I add them as ini options.